### ISSUE TITLE
No focus on fields

### ISSUE THREAD

**Issue by c1149** [microsoft/testfx-docs#86]
Since chromedriver https://chromedriver.storage.googleapis.com/index.html?path=87.0.4280.20/ i'm having issues with the tests running in parallel. 

Multiple browsers are opened, as expected. 
But only one browser session is clicking and inserting values. Only one page is opening the url I think. The other pages go to the url when I manually select the session. Nothing in our code has changed. 

Any ideas what it can be? Problem might have occured since V86. 

It is working on edgeChromium 86.0.622.68

**Comment by Evangelink** [comment]
@c1149 I will move forward and close this issue. Please if you are still having any issue, post a new issue on https://github.com/microsoft/testfx
